src/WorldState.Data/Models/Invasion.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Serialization;
+
+using WorldState.Data.Enums;
+
+namespace WorldState.Data.Models
+{
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy), ItemNullValueHandling = NullValueHandling.Ignore)]
+    public class Invasion
+    {
+        [JsonProperty]
+        public String Id { get; private set; }
```

Comment:
I'd also recommend following Microsoft's coding conventions regarding variable types:

`9. Do use predefined type names instead of system type names like Int16, Single, UInt64, etc.`

Reply:
I won't put a comment on each line that uses system type names though, your call if you want to change them all.

Reply:
Using alias is just a personal preference; it reminds me the capacity of the type (_Int16_ is 16-bits and _Single_ is single-precision, where _short_ and _float_ don't really indicate anything) and highlight on framework types look better in my editor 🤣 .

See related comments on "code analyzers".